source/usage-examples.txt

```diff
@@ -16,4 +20,43 @@ TODO: Write usage examples page
    /usage-examples/updateMany
    /usage-examples/replaceOne
    /usage-examples/deleteOne
-   /usage-examples/deleteMany
\ No newline at end of file
+   /usage-examples/deleteMany
+
+Overview
+--------
+
+Usage examples provide convenient starting points for popular MongoDB
+operations. Each example provides the following information:
+
+- Code snippets showing methods to perform the operation synchronously and
+  asynchronously
+
+- A fully runnable console application
```

Comment:
Issue: the phrasing of this list item doesn't flow well with the introductory text

```suggestion
- A link to a fully runnable console application using the operation
```